scipy/io/matlab/tests/test_mio.py

```diff
@@ -1055,6 +1055,27 @@ def test_unicode_mat4():
     assert_equal(var_back['second_cat'], var['second_cat'])
 
 
+def test_logical_sparse():
+    # Test we can read logical sparse stored in mat file as bytes.
+    # See https://github.com/scipy/scipy/issues/3539.
+    # In some files saved by MATLAB, the sparse data elements (Real Part
+    # Subelement in MATLAB speak) are stored with apparent type double
+    # (miDOUBLE) but are in fact single bytes.
+    filename = pjoin(test_data_path,'logical_sparse.mat')
+    # Before fix, this would crash with:
+    # ValueError: indices and data should have the same size
+    d = loadmat(filename, struct_as_record=True)
+    log_sp = d['sp_log_5_4']
+    assert_(isinstance(log_sp, SP.csc_matrix))
+    assert_equal(log_sp.dtype.type, np.bool_)
+    assert_array_equal(log_sp.toarray(),
+                       [[True,  True,  True, False],
```

Comment:
PEP8 is unhappy with the multiple spaces behind a comma on this line (error E241). In this case it doesn't hurt readability, but we have to please the bot...